### PR TITLE
[ENG-5804] - Mariia's GV local setup

### PR DESCRIPTION
**Pull request by ly-mariia**
- changed the order in the Dockerfile, docker didn't work because manage.py was run before installing requirements;
- added in README.md makemigrations command;
- changed the python3 command to python since we have one python;

### CODE REVIEW

File: addon_service/migrations/0002_alter_externalstorageservice_int_addon_imp_and_more.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            name="oauth2_client_config",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="external_storage_services",
+                to="addon_service.oauth2clientconfig",
+            ),
+        ),
+    ]
```
Comment (adlius) [inline addon_service/migrations/0002_alter_externalstorageservice_int_addon_imp_and_more.py:59]: Looks like we made some model changes without `makemigrations`. @jwalz @aaxelb Do we want to add a new migration file or do we want to fold it into `0001_initial`?

**Review by adlius — APPROVED** [review]
Thanks for updating the Dockerfile and the readme. Looks good.

File: addon_service/migrations/0002_alter_externalstorageservice_int_addon_imp_and_more.py (hunk 1 above)
Comment (aaxelb) [reply addon_service/migrations/0002_alter_externalstorageservice_int_addon_imp_and_more.py:59]: whoops, yeah i missed that

we are still updating `0001_initial` in place (`rm addon_service/migrations/000* && python manage.py makemigrations`), at least until we have external services set up on staging

File: Dockerfile [hunk 2]
```diff
@@ -6,15 +6,16 @@ RUN apt-get update && apt-get install -y libpq-dev
...
 # END gv-base
 
 # BEGIN gv-deploy
 FROM gv-base as gv-deploy
 # only non-dev dependencies:
 RUN pip3 install --no-cache-dir -r requirements/requirements.txt
 # note: no CMD in gv-deploy -- depends on deployment
+
+RUN python manage.py collectstatic --noinput
+EXPOSE 8000
```
Comment (aaxelb) [inline Dockerfile:18]: another whoops -- good catch, thx

**Review by aaxelb — COMMENTED** [review]
thanks for catching these mistakes! i've opened a pr with fixes at #68

**Comment by aaxelb** [comment]
fixes added with #68 -- thanks!

**Pull request closed without merging**